tests/switchless_threads/enc/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_custom_command(
+  OUTPUT switchless_threads_t.h switchless_threads_t.c switchless_threads_args.h
+  DEPENDS
+  edger8r
+  ../switchless_threads.edl
+  COMMAND edger8r --experimental --trusted --search-path ${CMAKE_CURRENT_SOURCE_DIR}/.. switchless_threads.edl)
+
+# Dummy target used for generating from EDL on demand.
+add_custom_target(switchless_threads_enc_gen DEPENDS switchless_threads_t.h switchless_threads_t.c switchless_threads_args.h)
+
+add_enclave(TARGET switchless_threads_enc UUID 6e818629-0ce7-46cd-822a-6c7e081fc68b SOURCES enc.c switchless_threads_t.c)
```

Comment:
`6e818629-0ce7-46cd-822a-6c7e081fc68b`

I am lost on where this value comes from and if it holds any significance.

```brmclare@BrettSGXDev:~/openenclave$ grep -rins 6e818629-0ce7-46cd-822a-6c7e081fc68b
tests/switchless_threads/enc/CMakeLists.txt:14:add_enclave(TARGET switchless_threads_enc UUID 6e818629-0ce7-46cd-822a-6c7e081fc68b SOURCES enc.c switchless_threads_t.c)```

It _seems_ to be completely random, looking at https://github.com/openenclave/openenclave/blob/master/cmake/add_enclave.cmake#L24 makes me wonder if the TA binaries will ever need a different UUID?

Reply:
That's a good question. Looking at the same cmake file, the UUID is used for signing the enclave binary. But even if it's not provided, the `add_enclave` will generate one on the fly. 

I generated the UUID manually and passed in here, just to follow the tradition of other tests. Do we really need to do so? @andschwa